sdk/src/Member.cs

```diff
@@ -1592,201 +1337,184 @@ public BankInfo GetBankInfoBlocking(string bankId)
         }
 
         /// <summary>
-        /// Applies SCA for the given a list of accounts.
+        /// Resolves transfer destinations for the given account.
         /// </summary>
-        /// <param name="accountIds">the list of accounts</param>
-        /// <returns>a task</returns>
-        public Task ApplySca(IList<string> accountIds)
+        /// <param name="accountId">the account id</param>
+        /// <returns>a list of transfer endpoints</returns>
+        public Task<IList<TransferEndpoint>> ResolveTransferDestinations(string accountId)
         {
-            return client.ApplySca(accountIds);
+            return client.ResolveTransferDestination(accountId);
         }
         
         /// <summary>
-        /// Applies SCA for the given a list of accounts.
+        /// Resolves transfer destinations for the given account.
         /// </summary>
-        /// <param name="accountIds">the list of accounts</param>
-        /// <returns>a task</returns>
-        public void ApplyScaBlocking(IList<string> accountIds)
+        /// <param name="accountId">the account id</param>
+        /// <returns>a list of transfer endpoints</returns>
+        public IList<TransferEndpoint> ResolveTransferDestinationsBlocking(string accountId)
         {
-            ApplySca(accountIds).Wait();
+            return ResolveTransferDestinations(accountId).Result;
         }
 
         /// <summary>
-        /// Signs a token request state payload.
+        /// **For testing purposes only**
+        /// Creates a linked test bank account.
         /// </summary>
-        /// <param name="tokenRequestId">the token request id</param>
-        /// <param name="tokenId">the token id</param>
-        /// <param name="state">the state</param>
-        /// <returns>the signature</returns>
-        public Task<Signature> SignTokenRequestState(
-            string tokenRequestId,
-            string tokenId,
-            string state)
+        /// <param name="balance">the account balance to set</param>
+        /// <returns>the OAuth bank authorization</returns>
+        public Task<ProtoAccount> CreateAndLinkTestBankAccount(Money balance)
```

Comment:
These were mentioned as separate functions in the excel file. There is no difference actually. CreateTestBankAccount calls CreateAndLinkTestBankAccount essentially. Should I remove one of them?

Reply:
The return is different, no? CreateTestBankAccount returns BankAuthorization that can be passed to LinkAccounts, whereas CreateAndLinkTestBankAccount actually *calls* linkAccounts and returns the linked account object.

The issue is that linkAccounts is not part of the TPP SDK so CreateTestBankAccount is not useful if it only returns BankAuthorization. Therefore it should return the linked account. Since that would duplicate the function of CreateAndLinkTestBankAccount, CreateAndLinkTestBankAccount should be removed.

Reply:
Removed  CreateAndLinkTestBankAccount